client/src/components/AlertDismissable/AlertDismissable.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types'
```

Comment:
New AlertDismissable React component that uses the [Bootstrap](https://react-bootstrap.github.io/components/alerts/) alert component